docs/OpenBSD.md

```diff
@@ -76,3 +67,38 @@ Install OpenBSD over PXE
   bsd.mp (multi-processor kernel)
   baseXX.tgz (OpenBSD base system) - essential
   ```
+
+5. In order to use Autoinstall mode, a few more adjustments need to be made.
+
+The binary you set to load in "filename" directive in DHCP server needs to be
```

Comment:
@pkubaj what if I do not have a DHCP server running?

Reply:
With this procedure, it won't work. I'll prepare another one, for when there's no DHCP.

Reply:
Please verify now.